configure.ac

```diff
@@ -455,6 +456,54 @@ AC_SUBST(MEEP_SUFFIX)
 program_transform_name="s,_,-,g;$program_transform_name"
 
 ##############################################################################
+# checks for python
+##############################################################################
+AC_ARG_WITH(python,
+	[AC_HELP_STRING([--without-python], [don't compile Python interface])],
+        with_python=$withval,with_python=yes)
+
+if test "$enable_shared" = no; then
+   AC_MSG_WARN([Python interface requires --enable-shared; disabling])
+   have_python=no
+else
+
+  if test "x$with_python" = xno; then
+    have_python=no
+  else
+    AM_PATH_PYTHON([],[have_python=yes],[have_python=no])
+    if test $have_python = yes; then
+
+      AC_MSG_CHECKING([for Python include directory])
+      pinc=`echo "import distutils.sysconfig; print (distutils.sysconfig.get_python_inc())" | $PYTHON - 2>/dev/null`
+      AC_MSG_RESULT([${pinc:-unknown}])
+      test -n "$pinc" && PYTHON_INCLUDES="-I$pinc"
+      save_CPPFLAGS=$CPPFLAGS
+      CPPFLAGS="$CPPFLAGS $PYTHON_INCLUDES"
+      AC_CHECK_HEADER([Python.h], [], [AC_MSG_WARN([disabling Python wrappers]) 
+                                       have_python=no])
+
+      if test $have_python = yes; then
+        AC_MSG_CHECKING([for Numpy include directory])
+        pinc=`echo "import numpy; print (numpy.get_include())" | $PYTHON - 2>/dev/null`
+        AC_MSG_RESULT([${pinc:-unknown}])
+        test -n "$pinc" && PYTHON_INCLUDES="$PYTHON_INCLUDES -I$pinc"
+        CPPFLAGS="$save_CPPFLAGS $PYTHON_INCLUDES"
+        AC_CHECK_HEADER([numpy/arrayobject.h],[],[
+          AC_MSG_WARN([disabling Python wrappers])
+          have_python=no],[#include <Python.h>])jl$A
+      fi
+
+      CPPFLAGS=$save_CPPFLAGS
```

Comment:
This appears to overwrite the `$PYTHON_INCLUDES` flags we just added, but my assumption is that we only need those flags for the calls to `AC_CHECK_HEADER` and the python includes get added at some later stage.  Is that correct?